src/com/facebook/buck/ide/intellij/lang/android/AndroidLibraryModuleRule.java

```diff
@@ -53,6 +53,7 @@ public void apply(
     Optional<Path> dummyRDotJavaClassPath = moduleFactoryResolver.getDummyRDotJavaPath(target);
     if (dummyRDotJavaClassPath.isPresent()) {
       context.addExtraClassPathDependency(dummyRDotJavaClassPath.get());
```

Comment:
this creates an extra library, but we need to create an extra module so that we can specify the compiler output (since that's where the Layout Preview loads classes from)